### DISCUSSION TITLE
Issue trying installing plugin for netbox 3.5.4

### DISCUSSION THREAD

**Discussion by Jayo-pham** [mattieserver/netbox-topology-views#331] [Q&A]
Hello,
Please help me with this issue, This is the first plugin that I'm trying to install.  I followed the instruction to the step where running migrate and receive error like the bottom line. 

![image](https://github.com/mattieserver/netbox-topology-views/assets/134136510/45fd52ed-98a4-441a-be07-d6a840815086)

![image](https://github.com/mattieserver/netbox-topology-views/assets/134136510/e0a25e77-47e5-4e51-9b9a-b37313a995fe)

**Comment by mattieserver** [comment]
you might need to run the migrate script with sudo aswell

**Reply by mattieserver** [reply]
You can also run the upgrade script `sudo /opt/netbox/upgrade.sh` (if you followed the install steps from the netbox docs), that script will also take care of all migrations.

**Reply by Jayo-pham** [reply]
Here's the configuration.py that I have
![Capture1](https://github.com/mattieserver/netbox-topology-views/assets/134136510/1427efeb-5f3d-4db9-84f9-f92f5f8ec321)

Here's the result when I ran sudo migrate

![Capture2](https://github.com/mattieserver/netbox-topology-views/assets/134136510/72c1e5ff-81ec-444f-ac14-89623a000de6)

Here's the error I get when trying the upgrade.sh

![Capture3](https://github.com/mattieserver/netbox-topology-views/assets/134136510/46ce920b-69b2-4d05-aa38-8113c083039d)

**Reply by dreng** [reply]
Seems to be not an issue with NetBox Topology Views. Your venv is missing setuptools, which should have been installed with NetBox as a dependency of gunicorn (see requirement.txt in NetBox root folder for gunicorn).

How did you install NetBox? ZIP, Git or Docker?

Please make sure you did everything exactly as [documented](https://docs.netbox.dev/en/stable/installation/).

**Reply by Jayo-pham** [reply]
i Installed netbox via cloning it from github and I followed every lines on Jeremy's instruction. so is that mean that I have to reinstall everything

**Reply by mattieserver** [reply]
That error seems strange, can you remove the plugin from the local requirements file and the config and run the upgrade script again? The script should always be able to succeed.
